main.go

```diff
@@ -2,7 +2,7 @@ package main
 
 import (
 	"fmt"
-	"github.com/urfave/cli"
+	 "github.com/bradrydzewski/cli"
 	"log"
```

Comment:
probably leave with urfave right now since this is vendored

Reply:
problem here is bradrydzewski has an additonal function to read files from disks so this might be trickier to resolve